web/02-modelsite.php

```diff
@@ -80,13 +80,154 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
 <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
 <link rel="stylesheet" type="text/css" href="sites.css" />
+<style>
+	body {font-size: 62.5%;}
+	label, input {display:block; padding-top: 10px;}
+	input.text { margin-bottom:12px; width:95%; padding:10px;}
+	fieldset { padding:0; border:0; margin-top:25px; }
+	h1 { font-size: 1.2em; margin: .6em 0; }
+	.ui-dialog .ui-state-error { padding: .3em; } 
+	.validateTips { border: 1px solid transparent; padding: 0.3em; } 
+	#container { display: table; }
+	#row { display: table-row; }
+	#left, #right, #middle { display: table-cell; padding-left: 10px; }
+</style>
 <script type="text/javascript" src="jquery-1.7.2.min.js"></script>
 <?php if (!$offline) {?>
 <script type="text/javascript" src="//www.google.com/jsapi"></script>
 <?php }?>
+<?php if (!$offline) {?>
+<script type="text/javascript" src="//code.jquery.com/jquery-1.10.2.js"></script>
+<?php }?>
+<?php if (!$offline) {?>
+<script type="text/javascript" src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+<?php }?>
+<?php if (!$offline) {?>
+<link rel="stylesheet" type="text/css" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css"/>
+<?php }?>
 <script type="text/javascript">
+
+  //$(function()  {
+  $(window).load(function() {
```

Comment:
can you clean up the following block to be more consistent with the next sections. In general I use 2 spaces for indentation. Also just 1 empty line between functions and remove some additional blank lines.